src/expr/src/explain/text.rs

```diff
@@ -800,13 +780,7 @@ impl fmt::Display for MirScalarExpr {
                 }
             }
             If { cond, then, els } => {
-                write!(f, "case when ")?;
-                cond.fmt(f)?;
-                write!(f, " then ")?;
-                then.fmt(f)?;
-                write!(f, " else ")?;
-                els.fmt(f)?;
-                write!(f, " end")
+                write!(f, "case when {} then {} else {} end", cond, then, els)
```

Comment:
We decided to adopt a style where scalar expressions are aligned with the SQL concrete syntax (as opposed to aligning them with rust, so: 
- `#0 AND #1` instead of `#0 && #1`,
- `#0 IS NOT NULL` instead of `not(is_null(#0))`, etc.
and I think is just to align with that.